helix-view/src/document.rs

```diff
@@ -162,6 +162,20 @@ impl Document {
         Ok(doc)
     }
 
+    pub fn reload(&mut self) -> Result<(), anyhow::Error> {
+        use std::{fs::File, io::BufReader};
+
+        let path = self
+            .path()
+            .ok_or(Error::msg("unable to reload document without path"))?;
+        let file = File::open(path).context(format!("unable to open {:?}", path))?;
```

Comment:
If will be an issue if the file doesn't exists.

Reply:
Wouldn't it just return whatever error `File::open` return given I ended with a `?`?